src/pyhf/tensor/tensorflow_backend.py

```diff
@@ -75,7 +75,14 @@ def tile(self, tensor_in, repeats):
             TensorFlow Tensor: The tensor with repeated axes
 
         """
-        return tf.tile(tensor_in, repeats)
+        try:
+            return tf.tile(tensor_in, repeats)
+        except tf.python.framework.errors_impl.InvalidArgumentError:
+            shape = tf.shape(tensor_in).numpy().tolist()
+            diff = len(repeats) - len(shape)
+            if diff < 0:
+                raise
+            return tf.tile(tf.reshape(tensor_in, [1] * diff + shape), repeats,)
```

Comment:
```suggestion
            return tf.tile(tf.reshape(tensor_in, [1] * diff + shape), repeats)
```

Not important, but could also grab that trailing comma left over from Black (one of Black's know bugs, if you want to call it one).